core/pom.xml

```diff
@@ -106,6 +106,12 @@
             <groupId>io.rest-assured</groupId>
             <artifactId>rest-assured</artifactId>
         </dependency>
+        <dependency>
```

Comment:
EM uses JUnit 5, not 4. why is this here?

Reply:
Honestly, I don't know. I guess the IDE simply added the dependency when I was starting to work with the project. I removed the dependency and it seems to work (all tests passed)